proposals/simd/SIMD.md

```diff
@@ -104,6 +104,26 @@ operations. This means that any subnormal operand is treated as 0, and any
 subnormal result is rounded to 0. Note that this differs from WebAssembly
 scalar floating-point semantics which require correct subnormal handling.
 
+# JavaScript API and SIMD Values
```

Comment:
It looks like you can go through the `js-api/index.bs` doc and search for `𝗂𝟨𝟦`, that has pretty much all the places you'll need to do the same thing.

The ones I see missing here:

* In `Global` constructor
* In `GetGlobalValue`
* In `Global.value` attribute

Reply:
Thanks, fixed.